src/session.ts

```diff
@@ -43,21 +43,13 @@ let activeBrowserExternalUri: Uri;
 export function deploySessionWatcher(extensionPath: string): void {
     console.info(`[deploySessionWatcher] extensionPath: ${extensionPath}`);
     resDir = path.join(extensionPath, 'dist', 'resources');
-    watcherDir = path.join(os.homedir(), '.vscode-R');
+    const tempPath = os.tmpdir();
+    watcherDir = path.join(tempPath, '.vscode-R');
     console.info(`[deploySessionWatcher] watcherDir: ${watcherDir}`);
     if (!fs.existsSync(watcherDir)) {
         console.info('[deploySessionWatcher] watcherDir not exists, create directory');
         fs.mkdirSync(watcherDir);
     }
-    console.info('[deploySessionWatcher] Deploy init.R');
```

Comment:
We put these files in a easily known directory because there's an important use case to start self-managed R sessions (.e.g in tmux window) where user sources the `init.R` scripts on their own.

If we don't put them in such an directory, user will have know the extension path to source the script, which does not seem to be friendly enough.

Reply:
That's definitely an important point, but I personally find it a bit confusing to have the extension copy all .R files around before running them, just so they are easier to find.

Having them in the extension's directory would make it clear that they are not meant to be modified (currently, any modifications are overwritten anyways). A one-line R script in `~/.vscode-R` that points to the extension's installation directory would work as well, and avoids having two identical copies of the same code.

Reply:
> A one-line R script in ~/.vscode-R that points to the extension's installation directory would work as well, and avoids having two identical copies of the same code.

I like this idea. But in this script, it seems that it has to also set an environment variable of the init script folder path so that `init.R` script could know the paths of other scripts, which is needed by self-managed R sessions.

Reply:
I added a file that contains only `local(source("/PATH/TO/init.R", chdir = TRUE, local = TRUE))`, which temporarily changes the wd to the dir containing `init.R` to make sure that local path references work in `init.R`.

Reply:
Ok, that's a neat trick.